spec/features/reply_from_councillor_spec.rb

```diff
@@ -1,10 +1,8 @@
 require "spec_helper"
 
 feature "Councillor replies to a message sent to them" do
-  given (:councillor) { create(:councillor, name: "Louise Councillor", email: "louise@council.nsw.gov.au") }
-  given (:application) { create(:application, id: 8, address: "24 Bruce Road Glenbrook", description: "A lovely house") }
-  given (:comment) do
   let(:councillor) { create(:councillor, name: "Louise Councillor", email: "louise@council.nsw.gov.au", popolo_id: "marrickville_council/chris_woods") }
+  let(:application) { create(:application, id: 8, address: "24 Bruce Road Glenbrook", description: "A lovely house") }
```

Comment:
Hey @davwood I think we did it this way to follow the [Capybara “descriptive” DSL](https://github.com/jnicklas/capybara#using-capybara-with-rspec). I kinda think it is nicer to be consistent through the tests and use let, before and describe here as you've changed—but we’ve used this DSL through the integration tests, so I think we should stick to it or change them all. What do you think?